src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/TvpTest.cs

```diff
@@ -50,7 +50,8 @@ public void TestMain()
             Assert.True(RunTestCoreAndCompareWithBaseline());
         }
 
-        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))] // [ActiveIssue(5531)]
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
+        [ActiveIssue(5531)]
```

Comment:
Was this intentional? We should keep running TestPacketNumberWraparound against box install instances unless it has started failing against them, too, now. (It was only intermittently failing against Azure SQL DB instances before.)

Reply:
As discussed in the standup today, I'll keep this active issue.